.npmignore

```diff
@@ -1,6 +1,4 @@
-src
```

Comment:
Erm...sorry about this, I was testing something on my repo and by mistake I also pushed this.